### ISSUE TITLE
Manage the app from a tablet

### ISSUE THREAD

**Issue by Marcurio22** [Marcurio22/GESPRO_Practica_2_Curso_2024_2025#10]

**Comment by aal1021** [comment]
¿La aplicación va a tener interfaces distintas para móvil y tablet?

**Comment by Marcurio22** [comment]
No, será la misma interfaz tanto para dispositivos móvil como para dispositivos tablet, ya que será una aplicación de Play Store y funcionará de la misma manera para cualquier dispositivo Android. Únicamente, en la tablet podrás ver el contenido de manera más ancha como es lógico.
